### ISSUE TITLE
A11Y

### ISSUE THREAD

**Issue by mischahaenen** [mischahaenen/nuenenen-frontend#16]
- [x] add datenschutz page 
- [x] display testimonials
- [x] breakout sections (Images full width)
- [x] fading-in elements in section
- [x] cookie consent
- [x] Check a11y (lighthouse and manual check)
- [x] camping svg
- [x] No contact on every page
- [x] Mobile Dropdown stays after clicking
- [x] Team to Pages
- [x] Document & Image for QP
- [x] Footer spacing reduce to 1rem
- [x] Images in Steps should be round
- [x] Step Detail should be round
- [x] Pagezone --> Aktivität
- [x] Slogan bigger text
- [x] Full width image slider

**Comment by mischahaenen** [comment]
https://www.w3.org/TR/WCAG20-TECHS/G1.html
